ddpui/utils/flags.py

```diff
@@ -0,0 +1,7 @@
+from flags import conditions
+
+
+@conditions.register("org")
+def org_condition(org_id, request_org_id=None):
```

Comment:
can we name the condition to `reigster("org_id")` ; so that someone updating things from admin panel knows it expects an id and not a org slug

do you think conditioning on `org_slug` would be more readable ? Just a thought, dont mind either.

Reply:
I think slug would be more readable. Updated it.